.travis.yml

```diff
@@ -35,3 +35,4 @@ deploy:
   on:
     repo: codercom/code-server
     branch: master
+cache: yarn
```

Comment:
These change were already pushed? I fail to see how this PR is related to a "Travis cache"